sync.sh

```diff
@@ -284,8 +288,9 @@ _start_new_loop() {
         printf "%b\n" "Job started.\nLocal Folder: ${INPUT}\nDrive Folder: ${GDRIVE_FOLDER}"
         printf "%s\n" "PID: ${PID}"
         printf "%b\n" "PID: ${PID}|:_//_:|${FOLDER}|:_//_:|${GDRIVE_FOLDER}" >> "${SYNC_LIST}"
-        { [[ -n ${SHOW_LOGS} ]] && tail -f "${LOGS}"; } || :
+        [[ -n ${SHOW_LOGS} ]] && tail -f "${LOGS}"
     fi
+    return 0HOME
```

Comment:
Codacy found an issue: [Can only return 0-255. Other data should be written to stdout.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=5787594)